docs/src/pages/css-in-js/api/api.md

```diff
@@ -142,8 +142,18 @@ const MyComponent = styled('div')({
   backgroundColor: 'red',
 });
 
+const MyThemeComponent = styled('div')(({
+  theme
+}) => {
```

Comment:
You probably want to return an object at which point you need to wrap it in parentheses..

Reply:
Even syntax highlight told me so. Sorry for the mistake.
Corrected now.

Reply:
Well it's valid syntax: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/label

I'll have the same issue fairly often too. It's one of the reasons why I'll try to avoid implicit return statements. The other being poor debuggability and that it is not supported for function declaration/expressions.